package.json

```diff
@@ -6,19 +6,50 @@
   "dependencies": {
     "a-sync-waterfall": "^1.0.0",
     "asap": "^2.0.3",
+    "postinstall-build": "^5.0.1",
```

Comment:
We can avoid this dependency for end-user by using `prepublishOnly` script, as described in my issue comment

Reply:
I don't think that's true if someone does a source install, e.g. `npm install mozilla/nunjucks`. The purpose of [postinstall-build](https://www.npmjs.com/package/postinstall-build#why) is to make `npm install` work with git installs without also having to move your build dependencies from `devDependencies` to `dependencies`

Reply:
That is not da way.

If someone crazy enough to make NPM install, he should realize consequences and should invoke building manually. That's is da way.

Adding another dependency to already bloated `node_modules` of any user is a cardinal sin. Besides, a module can be consumed not only by NPM, but also by JSPM, where postbuilding is simply unavailable (probably).

Reply:
I must not be explaining things properly. This 40K dependency (a 9K download from npm) makes it so that the build dependencies are _not_ needed to npm install the package. Failure to do something like this would break `npm install mozilla/nunjucks` (or, more likely `npm install userwithfork/nunjucks`) which is a pretty common use-case. Otherwise I would need to commit the dist files to source control (bad) or move the build deps from `devDependencies` to `dependencies` (worse).

Reply:
Ah, now I finally got what you're trying to do. I've read `postinstall-build` description and got an impression that you want completely move prepublishing build to the end-user. My bad.

Well, I see the point. Not sure that it worth it, but anyway, why not.